templates/report.html

```diff
@@ -252,9 +267,35 @@ <h1 id="kyoku-{{ item.kyoku }}-{{ item.honba }}" class="kyoku-heading">
           {%- endif -%}
         </details>
       {%- endfor -%}
+      </div>
     </section>
   {%- endfor -%}
 
+  <script>
+    function toggleStyle() {
+      if (document.getElementById("radio_style_v").checked) {
+        document.body.removeAttribute("horizontal")
+        lboxes = document.querySelectorAll(".l-box")
+        for (var i = 0; i < lboxes.length; i++) {
+          lboxes[i].removeAttribute("horizontal")
+        }
+        rboxes = document.querySelectorAll(".r-box")
+        for (var i = 0; i < rboxes.length; i++) {
+          rboxes[i].removeAttribute("horizontal")
+        }
+      } else {
+        document.body.setAttribute("horizontal", "1")
+        lboxes = document.querySelectorAll(".l-box")
+        for (var i = 0; i < lboxes.length; i++) {
+          lboxes[i].setAttribute("horizontal", "1")
+        }
+        rboxes = document.querySelectorAll(".r-box")
+        for (var i = 0; i < rboxes.length; i++) {
+          rboxes[i].setAttribute("horizontal", "1")
+        }
+      }
+    }
+  </script>
```

Comment:
Maybe create a separate standalone javascript template file and `include` it here? Just like the CSS.

Reply:
done